ios/Classes/SwiftFlutterWebAuthPlugin.swift

```diff
@@ -46,13 +46,31 @@ public class SwiftFlutterWebAuthPlugin: NSObject, FlutterPlugin {
                 let session = ASWebAuthenticationSession(url: url, callbackURLScheme: callbackURLScheme, completionHandler: completionHandler)
 
                 if #available(iOS 13, *) {
-                    guard let provider = UIApplication.shared.delegate?.window??.rootViewController as? FlutterViewController else {
-                        result(FlutterError(code: "FAILED", message: "Failed to aquire root FlutterViewController" , details: nil))
+                    if var topController = UIApplication.shared.keyWindow?.rootViewController {
```

Comment:
Since this `if` statement has a returning else condition, could you change it to a `guard`?